src/System.Private.ServiceModel/tests/Scenarios/Binding/WS/FederationHttp/WSFederationHttpBindingTests.cs

```diff
@@ -32,7 +32,6 @@ public static void WSFederationHttpBindingTests_Succeeds()
         {
             // *** SETUP *** \\
             issuerAddress = new EndpointAddress(new Uri(Endpoints.WSFederationAuthorityLocalSTS));
-            tokenTargetAddress = Endpoints.Https_SecModeTransWithMessCred_ClientCredTypeIssuedTokenSaml2;
             serviceEndpointAddress = new EndpointAddress(new Uri(tokenTargetAddress));
```

Comment:
This broke the test as it's used on the next line to create the serviceEndpointAddress.

Reply:
fixed in: 9a8f277d85bf9419167870cad953b59d0d7a6559